frontend/src/org/pages/CloseEvent/CloseEvent.tsx

```diff
@@ -209,11 +209,14 @@ export const CloseEvent = () => {
       ...deletedReceiptPromises,
     ])
 
-    showMessage({
-      type: 'success',
+    showMessage({ 
+      type: evidence.is_closed ? 'success' : 'warning',
       message:
         'Evidence akce byla úspěšně uložena' +
-        (evidence.is_closed ? ' a uzavřena' : ''),
+        (evidence.is_closed
+          ? ' a uzavřena'
+          : '. Nezapomeň akci ještě uzavřít! Akci uzavřeš kliknutím na tlačítko "uložit a uzavřít" do 20 dnů od skončení akce.'),
+      timeout: evidence.is_closed ? 5000 : 10_000
```

Comment:
Is there really supposed to be this symbol "_"? What does it mean?

Reply:
Yes, it is just to make it easier to read the number, with the same meaning as '10000'

Reply:
Ah, ok! Interesting.